### ISSUE TITLE
how much LCD can be connect in one ESP32?

### ISSUE THREAD

**Issue by koasi** [kitecraft/Multiple_TFT_Displays#4]
as your project, you had used one ESP32 to control 3 panels . 
is that possible to control 8 panels via one ESP32?

**Comment by Nikhil2024me** [comment]
yes, it is possible to do define each cs of the tft_display like a new cs pin

**Comment by kitecraft** [comment]
Actually, no.  In this old library I only define 4 bits for LED selection.  Not 8.  Lazy, I know.
